### PR TITLE
Milestone Using Bootstrap

### PR DESCRIPTION
**Pull request by emiliocm9**
# Web Using Bootstrap
This PR address the completion of the project based on the design which is to build a clone of the Newsweek main web page, using Twitter's Bootstrap to style its content. 

- Finished the design using Bootstrap to style most of the content of the webpage.

- Selected Font for the design.

- Use svg code to add the company logo on the web page.

### CODE REVIEW

**Comment by sumancrest0001** [comment]
**STATUS: CHANGES REQUIRED
Fabulous work @emiliocm9 :clap:  :clap:** 
The project looks very close to the original site. since it is a clone project, we have to fix a couple of minor issues to get :+1: :+1:  from our side. Kindly make the following changes and submit them for another review.
[Required] On the medium screen, the signup section should not cover the whole page and should be centered. Currently, it is covering the whole screen. Please have a look.
Screenshot from 2020-04-17 09-39-11

It should be :point_down: 
Screenshot from 2020-04-17 09-38-59

**Comment by sumancrest0001** [comment]
[Required] On medium screen, please add the same margin between image and text section on the newsletter and subscribe section. 
It is currently :point_down: 
Screenshot from 2020-04-17 09-40-40

It should be like this :point_down: 

Screenshot from 2020-04-17 10-27-56

Also, align the titles.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,995 @@
...
+                <span class="second__heroes-t">"I want to keep it going as long as we can," Ryan O'Connell told Newsweek.</span>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="first__column-title text-uppercase font-weight-bold mt-3">Culture & Travel</div>
+        <div class="container">
+          <div class="row">
+            <div class="col-sm-4 col-12 pr-3 pl-0">
+              <div class="second__heores-icontainer position-relative">
```
Comment (sumancrest0001) [inline index.html]: [Required] on a small screen, Culture and travel section should be one column layout but currently, it is three columns layout. 
in your project :point_down: 
Screenshot from 2020-04-17 09-50-08

it should be :point_down: 
Screenshot from 2020-04-17 09-50-48

Comment (sumancrest0001) [reply index.html]: Make a similar layout for the heroes of the pandemic section also.

**Comment by sumancrest0001** [comment]
[Require] Please make sure that image sizes are similar to that of the original one (on small and medium screens).
Currently,
Screenshot from 2020-04-17 10-52-43

It should be :point_down: 
Screenshot from 2020-04-17 09-46-34

**Top story** should be at the top on a small screen but on the current layout, it is after the **featured stories.**

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
Please submit for another review once you made the requested changes.
Have questions? DM me on slack **Suman Shrestha**
**Happy coding**

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved 🏆 🎈 

Hi @emiliocm9, congratulations on fixing all the recommended changes pointed out by the previous TSE 👍 Great job, keep up the good work 🙂 
Go ahead and merge to master and do not forget to submit the project completion form 👍 

Good luck with your next projects 😄 
_Happy coding_ 👨‍💻 
Daniel Larbi Addo on slack

**Pull request merged**